src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -118,5 +121,21 @@ void Navigate(Frame frame)
 				frame.Navigate(typeof(Properties), argument, new SuppressNavigationTransitionInfo());
 			}
 		}
+
+		private static string GetFilesLogoPath()
+		{
+			var appTilesPath = Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Dev")))
+				return Path.Combine(appTilesPath, "Dev", "Logo.ico");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Preview")))
+				return Path.Combine(appTilesPath, "Preview", "Logo.ico");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Release")))
+				return Path.Combine(appTilesPath, "Release", "Logo.ico");
+
+			throw new InvalidOperationException("Cannot find Logo.ico from Assets/AppTiles.");
```

Comment:
In case both the path and icon name are extracted into the constants, using their values inside the exception message would be a good idea.

Reply:
What path? This happens when none of the 3 exist.

Reply:
I meant the `Assets/AppTiles` part of the path, sorry for the unclear comment.